src/routes/messages/messages.controller.spec.ts

```diff
@@ -946,6 +949,8 @@ describe('Messages controller', () => {
           message: 'Expected object, received number',
         });
     });
+
+    it.todo('should disable eth_sign');
```

Comment:
This will be included after https://github.com/safe-global/safe-client-gateway/pull/2422 is merged.